Cargo.toml

```diff
@@ -40,6 +40,47 @@ default-members = [
     "libparsec/crates/types",
 ]
 
+[workspace.dependencies]
```

Comment:
[According to documentation](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table), the dependencies in the workspace cannot be marked as optional.

Does this means the crates are downloaded/compiled no matter what ? Can this pose an issue in case of a crate that doesn't support a given platform ?

My guess is there is no issue there, but I'd rather rely on actual proof ^^
I theory web/android related crates can compile on non-web platform (otherwise rust-analyzer would not work with them), so this may not be an issue anyway...

Reply:
:+1: